main.js

```diff
@@ -1,11 +1,28 @@
+var $users = document.querySelector("#container-users")
+var $teacher = document.querySelector("#teacher-container")
+var $student = document.querySelector("#student-container")
+var $parent = document.querySelector("#parent-container")
+var $userBtn = document.querySelectorAll(".menu-btns")
 
 
-function hide() {
-  var $users = document.querySelector("#container-users")
-  $users.classList.add("hidden");
+
+function hide(event) {
+  $users.classList.toggle("hidden")
+  var $id = event.target.getAttribute("id")
```

Comment:
We stick to `$` prefixes for HTML elements, but not for other values like strings.